src/game/rendering_graph_node.c

```diff
@@ -248,6 +252,83 @@ Mtx identityMatrixWorldScale = {{
      0x00000000,                            LOWER_FIXED(1.0f)               <<  0}
 }};
 
+static void lists_render(Gfx **ptempGfxHead, struct DisplayListNode* currList)
+{
```

Comment:
Nitpick: Function brackets should be on the same line for repo consistency

Reply:
Fixed, I am too used to allman :)